### DISCUSSION TITLE
Will I lose the authenticated users if I change the Google Cloud project for "Login with Google"

### DISCUSSION THREAD

**Discussion by prateeksachan** [supabase/supabase#30479] [Questions]
Hello,

I've currently added the Google Provider for `Login with Google` but now have to shift to a different Google Cloud project. So, the `Client IDs` & `Client secrets` will have to be changed.

Will this have any issue with the users that I have in supabase or any authentication issues for them?

**Comment by GaryAustin1** [comment]
If the emails for the users are returned the same then it should work for the Supabase side.  You can signin with multiple providers such as github and discord as long the email is the same.
